Add support for Elfeed

drewdiver opened the pull request:
Hopefully other Elfeed users will find this addition useful.

drewdiver committed:
Add support for Elfeed

john2x reviewed (changes requested):
Thanks for the PR. For consistency, can you change the placement of the unquote (`,`) to right before the symbol?

e.g.
`(:foreground, blue)` -> `(:foreground ,blue)`

drewdiver committed:
Update plan9-theme.el

Updated the placement of the unquote (,) to right before the symbol for Elfeed.

drewdiver commented:
All set
